Kickstarter-iOS/Views/CreatePasswordFooterView.swift

```diff
@@ -0,0 +1,23 @@
+import Prelude
+import Library
+
+internal final class CreatePasswordFooterView: UITableViewHeaderFooterView, NibLoading {
```

Comment:
I forgot to mention that since this is a pretty generic view I might be able to re-use it very very soon. Would you mind renaming this to `SettingsHeaderView` and injecting the `descriptionLabel` text instead (through `configure(with email: String, description: String)`)? 🙏

Reply:
It sounds good! 👍